app/pages/sections/service-recipients/template.njk

```diff
@@ -29,20 +30,32 @@
         })}}
 
         <div class="nhsuk-grid-row nhsuk-u-font-size-19 nhsuk-u-font-weight-bold">
-          <div data-test-id="organisation-heading" class="nhsuk-grid-column-one-half">
+          <div data-test-id="organisation-heading" class="nhsuk-grid-column-two-thirds">
             {{ organisationHeading }}
           </div>
-          <div data-test-id="ods-code-heading" class="nhsuk-grid-column-one-half">
+          <div data-test-id="ods-code-heading" class="nhsuk-grid-column-one-third">
             {{ odsCodeHeading }}
           </div>
         </div>
       </div>
     </div>
 
-    <hr class="bc-c-order-table--hr nhsuk-grid-column-full nhsuk-u-margin-top-4 nhsuk-u-margin-bottom-4"/>
+    <hr class="bc-c-order-table__hr nhsuk-grid-column-full nhsuk-u-margin-top-4 nhsuk-u-margin-bottom-4"/>
 
     <div class="nhsuk-grid-row">
-      <div class="nhsuk-grid-column-two-thirds">
+      <div class="nhsuk-grid-column-two-thirds nhsuk-u-font-size-16">
+        {% for row in tableData %}
+          <div class="nhsuk-grid-row">
+            <div data-test-id="organisation-name-checkbox-{{row.odsCode}}" class="nhsuk-grid-column-two-thirds">
+              {{ checkboxes({
+                  items: [row.organisationName]
+                }) }}
+            </div>
+            <div data-test-id="ods-code-{{row.odsCode}}" class="nhsuk-grid-column-one-third bc-c-service-recipients-table__ods">
+              {{ row.odsCode }}
+            </div>
+          </div>
+        {% endfor %}
```

Comment:
I thought we were going to use the table component we have here?

Reply:
its because it line rows in between every row
and no other component would have a table with no lines

Reply:
Sorry I don't quite understand that. I thought we had proven out that our table component would be able to handle this sort of data coming in

Reply:
we checked with Lorraine and it is the only table that looks like this so rather than messing around with the component we just went with the easiest. option of rendering it in 2 columns like this

Reply:
it will accept the checkboxes fine but puts lines between each one

Reply:
ok couldn't we just update the component to pass a flag so them lines are not shown. We've gone through the trouble of creating the component. We may as well use it.

Reply:
we are going to use it latee with the input fields - it is more hassle than it is worth to put that flag in 😄

Reply:
tech debt?